requirements/development.txt

```diff
@@ -134,10 +136,19 @@ sphinxcontrib-serializinghtml==2.0.0
     # via sphinx
 sqlparse==0.5.1
     # via django
+tomli==2.0.1
+    # via
+    #   build
+    #   django-stubs
+    #   mypy
+    #   nox
+    #   pytest
+    #   sphinx
```

Comment:
While some of the upgrades on this file look alright, these new libraries seemed a bit suspect, so I went looking.

`tomli` is only required for Python < 3.11 see [[1](https://github.com/wntrblm/nox/blob/main/pyproject.toml#L45)][[2](https://github.com/sphinx-doc/sphinx/blob/master/pyproject.toml#L73)][[3](https://github.com/pytest-dev/pytest/blob/main/pyproject.toml#L55)]

This package is not required any more in recent versions of Python as [PEP 680](https://peps.python.org/pep-0680/) has added tomli to the standard library on Python 3.11.

I think it is possible that you are using an older version of Python and might need to update your Python binary.
That should be the reason why these updates are necessary on your machine.

For the purposes of these files in the master branch, I think we should always keep them aligned with the latest versions of Python we support, in this case Python 3.12.